drivers/mrf24j40/include/mrf24j40_registers.h

```diff
@@ -219,6 +219,28 @@ extern "C" {
 
 /** @} */
 
+/**
+ * @name    Shift offsets for TXMCR register (0x11)
```

Comment:
```suggestion
 * @brief   Shift offsets for TXMCR register (0x11)
```
should make CI happy